### PR TITLE
Add version information to dartdoc footer

### LINKED ISSUE #10787: Flutter version on docs.flutter.io
Here is the documentation footer on docs.flutter.io 
It is the most recent but there's no human readable version to follow-up. 

`Flutter •  Dart • cc license • 2017-06-15 01:01 • 6f77b4a9a76e57c4e6d23e61ba59a0cdd80d22b7`

@sethladd Edit if required

### PR DESCRIPTION
**Pull request by jcollins-g**
Fix #10787 by adding the version field to the generated pubspec.yaml.

### CODE REVIEW

**Comment by sethladd** [comment]
Thanks!

**Comment by devoncarew** [comment]
@jcollins-g, what does the footer look like w/ this change? I suspect that we'll insert the version, date, and git hash, which may be a bit much. We may want to drop one or more of those.

File: dev/tools/dartdoc.dart [hunk 1]
```diff
@@ -29,9 +30,12 @@ Future<Null> main(List<String> args) async {
   if (path.basename(Directory.current.path) == 'tools')
     Directory.current = Directory.current.parent.parent;
 
+  final RawVersion version = new RawVersion('VERSION');
+
   // Create the pubspec.yaml file.
   final StringBuffer buf = new StringBuffer('''
 name: Flutter
+version: ${version.toString()}
```
Comment (devoncarew) [inline dev/tools/dartdoc.dart]: Just `$version` should be enough here.

Comment (jcollins-g) [inline dev/tools/dartdoc.dart]: done.

**Comment by jcollins-g** [comment]
The footer does indeed include all of those, now.  But I don't know if it is helpful or not to continue to have the git tag in the presence of the version.  It probably is for bleeding edge but less so elsewhere.  I'll let reviewers weigh in.

screenshot from 2017-06-19 15 33 14

**Comment by devoncarew** [comment]
It might be nice to to drop the git hash for git branches == `alpha` (and for non-alpha docs we could shorten the hash to the first 10 chars since that's likely unique) - but not necessary for this PR.

lgtm

**Pull request merged**